docs/python/testing.md

```diff
@@ -12,144 +12,27 @@ MetaSocialImage: images/tutorial/python-social.png
-
-![VS Code prompt to install a test framework when enabled](images/testing/install-framework.png)
+If you enable pytest, but it is not currently installed in the activated environment, the Python extension will attempt to install it in the background. Furthermore, if both frameworks are enabled, the Python extension will only run `pytest`.
 
 > **Note**: The minimum supported version of pytest for the python extension is 7.0.0.
 
-## Create tests
-
-Each test framework has its own conventions for naming test files and structuring the tests within, as described in the following sections. Each case includes two test methods, one of which is intentionally set to fail for the purposes of demonstration.
-
-### Tests in unittest
-
-Create a file named `test_unittest.py` that contains a test class with two test methods:
-
-```python
-import inc_dec    # The code to test
-import unittest   # The test framework
-
-class Test_TestIncrementDecrement(unittest.TestCase):
-    def test_increment(self):
-        self.assertEqual(inc_dec.increment(3), 4)
-
-    # This test is designed to fail for demonstration purposes.
-    def test_decrement(self):
-        self.assertEqual(inc_dec.decrement(3), 4)
-
-if __name__ == '__main__':
-    unittest.main()
-```
-
-### Tests in pytest
-
-Create a file named `test_pytest.py` that contains two test methods:
-
-```python
-import inc_dec    # The code to test
-
-def test_increment():
-    assert inc_dec.increment(3) == 4
-
-# This test is designed to fail for demonstration purposes.
-def test_decrement():
-    assert inc_dec.decrement(3) == 4
-```
-
 ## Test discovery
 
 By default, the Python extension attempts to discover tests once you enable a framework. You can also trigger test discovery at any time using the **Test: Refresh Tests** command from the Command Palette.
 
-`python.testing.autoTestDiscoverOnSaveEnabled` is set to `true` by default, meaning that test discovery is also performed automatically whenever you add, delete, or update any Python file in the workspace. To disable this feature, set the value to `false`. You can refine for which files auto test discovery occurs by specifying a glob pattern in the `python.testing.autoTestDiscoverOnSavePattern` setting. Its default value is `**/*.py`.
-
-You can [configure the settings](/docs/configure/settings.md) in the Settings editor or directly in the `settings.json` file. You need to reload the window for the test discovery setting to take effect.
+> **Tip**: `python.testing.autoTestDiscoverOnSaveEnabled` is set to `true` by default, meaning that test discovery is also performed automatically whenever you add, delete, or update any Python file in the workspace. To disable this feature, set the value to `false`, which can be done either in the Settings editor or in the `settings.json` file as described in the VS Code [Settings](/docs/editor/settings.md) documentation. You will need to reload the window for this setting to take effect.
```

Comment:
think we should still include this section: `python.testing.autoTestDiscoverOnSavePattern` setting. Its default value is `**/*.py`.